environment.doc.yml

```diff
@@ -5,9 +5,14 @@ channels:
 dependencies:
     - numpydoc=1.1.0
     - pandas=1.3.5
+    - pip=22.0.3
+    # - pip:
+    #     - scikeras==0.4.1
     - python=3.8
     - scikit-learn=1.0.1
-    - sphinx=4.3.2
+    # - sphinx=4.3.2
     - sphinx-gallery=0.10.1
-    - sphinx_rtd_theme=1.0.0
+    # - sphinx_rtd_theme=1.0.0
+    # - tensorflow=2.7.0
```

Comment:
Don't we need these for the Cifar10 notebook?

Reply:
No we don't. The notebooks are run with the `mapie-notebooks` environment and can be run directly on colab simply by installing MAPIE.

Reply:
I removed the unnecessary librairies.